stories/components/color.stories.js

```diff
@@ -0,0 +1,36 @@
+import { h } from 'preact';
+import { Button } from '@Components/Button';
+
+export default {
+  title: 'Design System/Styles/Colors',
+  component: Button,
+  argTypes: {
+    onClick: {
+      action: 'clicked'
+    },
+    variant: {
+      control: {
+        type: 'select',
+        options: [
+          "black",
+          "grey-1",
+          "grey-2",
+          "grey-3",
+          "grey-4",
+          "grey-5",
+          "grey-6",
+          "danger",
+          "success"
+        ]
+      }
+    }
+  },
+};
+
+const Template = (args) => <label className={`color-${args.variant}`} >Helloworld</label>;
```

Comment:
Are we showing a `Button` (see line 6) or a `label`?

Reply:
It is a label, but there is no corresponding component you can use in code.

Are you familiar with storybook? (I'm not),
according to the doc the component parameter is required,
I'm not sure what's the right value to put there.

https://storybook.js.org/docs/react/api/csf#default-export

Reply:
> It is a label, we're demo'ing styles here, but there is no corresponding component you can use in code.

Then how about creating a component just for demo'ing? Could be something like:

```javascript
const DemoComponent = ({ variant, text }) => <label className={`color-${variant}`}>{text}</label>;
```

Then, you can pass it to the `component` parameter and create a template with it.

Reply:
Sounds good, I've used the DemoComponent from the story itself (don't want to burden the core with demo code)